src/Behat/Testwork/Tester/ServiceContainer/TesterExtension.php

```diff
@@ -137,6 +143,23 @@ protected function loadExerciseController(ContainerBuilder $container, $skip = f
         $definition->addTag(CliExtension::CONTROLLER_TAG, array('priority' => 0));
         $container->setDefinition(CliExtension::CONTROLLER_TAG . '.exercise', $definition);
     }
+    
+    
+    /**
+     * Loads stop on failure controller.
+     */
+    private function loadStopOnFailureHandler(ContainerBuilder $container, ?bool $stopOnFailure)
+    {
+        $definition = new Definition('Behat\Testwork\Tester\Handler\StopOnFailureHandler', array(
+            new Reference(EventDispatcherExtension::DISPATCHER_ID)
+        ));
+        $definition->addMethodCall('setResultInterpreter', array(new Reference(TesterExtension::RESULT_INTERPRETER_ID)));
```

Comment:
You can probably just set this in the constructor instead of having to do a separate call

Reply:
Yes, probably due to all the changes 😉 
This has been changed